<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PCLViewer</class>
 <widget class="QMainWindow" name="PCLViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>967</width>
    <height>806</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>5000</width>
    <height>5000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PCLViewer</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QVTKWidget" name="qvtkWidget">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>640</width>
      <height>711</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="CalculateKeypoints">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>640</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Calculate 
Keypoints</string>
    </property>
   </widget>
   <widget class="QLabel" name="CubeEdgeLength">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>CubeEdgeLength</string>
    </property>
   </widget>
   <widget class="QLabel" name="CubeEdgePoints">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>70</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>CubeEdgePoints</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="CubeEdgePointsBox">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>100</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="decimals">
     <number>0</number>
    </property>
    <property name="maximum">
     <double>1000.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>10.000000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="defaultValues">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>640</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Default
Values</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputModelBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>113</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>Give a valid .pcd file to load the model from</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>PCD Input File</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="UsePcdInputBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Use PCD Input File</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="DownSampleBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>97</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Downsample</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="downSampleLeafSizeBox">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>160</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="decimals">
     <number>5</number>
    </property>
    <property name="singleStep">
     <double>0.001000000000000</double>
    </property>
    <property name="value">
     <double>0.010000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="CubeEdgeLengthBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>62</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="decimals">
     <number>3</number>
    </property>
    <property name="maximum">
     <double>100.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>0.400000000000000</double>
    </property>
   </widget>
   <widget class="QCheckBox" name="refineKeypointsBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Refine Keypoints</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>260</y>
      <width>291</width>
      <height>16</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="nonMaxNormalRadiusBox">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>290</y>
      <width>62</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="decimals">
     <number>5</number>
    </property>
    <property name="value">
     <double>0.010000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="thresholdBox">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>330</y>
      <width>62</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="decimals">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLabel" name="nonMaxNormalRadius">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>300</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>nonMax/Normal Radius</string>
    </property>
   </widget>
   <widget class="QLabel" name="Threshold">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>340</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Threshold</string>
    </property>
   </widget>
   <widget class="QComboBox" name="responseMethodBox">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>370</y>
      <width>78</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>HARRIS</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>NOBLE</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>LOWE</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>TOMASI</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CURVATURE</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="responseMethod">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>380</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Response Method</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="useMLSBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Use Moving Least Squares</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="mlsSearchRadiusBox">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>220</y>
      <width>62</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="decimals">
     <number>4</number>
    </property>
    <property name="value">
     <double>0.030000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>140</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>downsample leafsize</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>200</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>MLS SearchRadius</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
